### ISSUE TITLE
Nothing running

### ISSUE THREAD

**Issue by stevevaius77** [pavsa/hackrf-spectrum-analyzer#1]
After running, in first try it worked but then decided to change Frequency start and end points it stopped to work. After several times running nothing working still. Just GUI there with no activity

![image](https://cloud.githubusercontent.com/assets/24809488/23869668/7f1fe2de-082c-11e7-9791-28229ea66bd2.png)

**Comment by pavsa** [comment]
Reset the hackrf with the button if that happens. For some reason, hackrf sometimes stops sending data. It seems to be an issue with the firmware hackrf itself. I have yet to find solution for this.

**Comment by stevevaius77** [comment]
Yeah, resetting it worked, just found before your reply. Looking fwd to hear improvements

**Comment by pavsa** [comment]
I have tested it and it does the same thing with the linux utility hackrf_sweep, so I guess I'll open a bug report there.
